enterprise/server/util/cpuset/cpuset.go

```diff
@@ -75,60 +75,58 @@ func Format[I constraints.Integer](cpus ...I) string {
 	return strings.Join(cpuStrings, ",")
 }
 
-func parseListFormat(s string) ([]cpuInfo, error) {
-	// Example: "0-1,3" is parsed as []int{0, 1, 3}
-	var nodes []cpuInfo
-	var physicalID int
+// parseCPUs parses a string like "0,1-2" or "0:0-127,1:128-255" to cpuInfos.
```

Comment:
Could you add a comment on valid values:
Ex. 0:0-127 // node with cpu range
0-127 // no node, cpu range
0 // no node, only start of cpu range

Reply:
Done